docs/content/spark/procedures.md

```diff
@@ -109,6 +109,18 @@ This section introduce all available spark procedures about paimon.
          CALL sys.create_tag_from_timestamp(`table` => 'default.T', `tag` => 'my_tag', `timestamp` => 1724404318750, time_retained => '1 d')
       </td>
     </tr>
+    <tr>
+      <td>rename_tag</td>
+      <td>
+         Rename tag to new tag name. Arguments:
+            <li>table: the target table identifier. Cannot be empty.</li>
+            <li>tag_name: name of the tag. Cannot be empty.</li>
+            <li>target_tag_name: the new tag name to rename.</li>
```

Comment:
target_tag_name: the new tag name to rename. Cannot be empty.

Reply:
done